percona/pmm/pmm.go

```diff
@@ -229,10 +233,10 @@ func sidecarContainerV2(pgc *v2.PerconaPGCluster) corev1.Container {
 		},
 	}
 
-	if pgc.CompareVersion("2.3.0") >= 0 {
+	if pgc.CompareVersion("2.7.0") >= 0 {
```

Comment:
We need to update our pmm unit tests that were introduced with the PMM3 support and use version 2.7 on the following as well:

https://github.com/percona/percona-postgresql-operator/blob/94252cc2e2f591a426afbd81da58a7a66a73e171/percona/pmm/pmm_test.go#L153